server/controllers/chartsController/chartsController.test.js

```diff
@@ -109,6 +112,65 @@ describe('chartsController', () => {
           ],
         })
       })
+
+      it('returns a list of charts that match the search query, the results are sorted by favorites first then title', async () => {
+        const [chart1, chart2, chart3] = [
+          createChart({
+            title: 'recent chart',
+            assessment: 'EEG',
+            variable: 'Rating',
+            owner: 'user',
+          }),
+          createChart({
+            title: 'EEG',
+            assessment: 'EEG Drive',
+            variable: 'drtive',
+            owner: 'user',
+            title: 'all charts',
+          }),
+          createChart({
+            title: 'recent chart',
+            assessment: 'Foo',
+            variable: 'Rating',
+            owner: 'anotherUser',
+            sharedWith: ['user'],
+          }),
+        ]
+        const { user, anotherUser } = {
+          user: createUser({
+            uid: 'user',
+            favoriteCharts: ['5'],
```

Comment:
is this an actual ID of a chart? I don't see it specified anywhere.

Reply:
[216b66e](https://github.com/AMP-SCZ/dpdash/pull/560/commits/216b66e967a28fae64661657855e58bd0984fef3)
yes and I hardcoded it, now it should make more sense.